src/app.js

```diff
@@ -27,6 +27,7 @@ var resultImageElSmDk = document.getElementById('result-image-sm-dk');
 var resultImageElXsDk = document.getElementById('result-image-xs-dk');
 var resultSectionEl = document.querySelector('.result');
 var resultInstructionEl = document.getElementById('result-instruction');
+var darkModeToggle = document.getElementById('dark-theme-toggle');
```

Comment:
BTW, I am cool with some DOM abstraction like jQuery, d3-selection, or even `var sel = document.querySelector;` if you're tired of typing all this out. I just skipped all that because I didn't want to get distracted initially!